src/containers/Accounts/AMM/AMMAccounts/AMMAccountHeader/AMMAccountHeader.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import '../../../../shared/css/nested-menu.scss'
+import '../../../AccountHeader/styles.scss'
+import '../../../AccountHeader/balance-selector.scss'
+import { localizeBalance, localizeNumber } from '../../../../shared/utils'
+import Currency from '../../../../shared/components/Currency'
+
+export interface AmmDataType {
+  balance: { currency: string; amount: number; issuer: string }
+  balance2: { currency: string; amount: number; issuer: string }
+  lpBalance: number
+  tradingFee: number
+  accountId: string
+  language: string
+}
+
+export const AMMAccountHeader = (props: { data: AmmDataType }) => {
+  const { data } = props
+  const { balance, balance2, tradingFee, lpBalance, accountId, language } = data
+  const { t } = useTranslation()
+  const localizedBalance1 = localizeBalance(balance, language)
+  const localizedBalance2 = localizeBalance(balance2, language)
+  const tradingFeeTotal = 1000
+  const localizedLPBalance = localizeNumber(lpBalance, language, {
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 2,
+  })
+  const localizedTradingFee = localizeNumber(
+    tradingFee / tradingFeeTotal,
+    language,
+    {
+      minimumFractionDigits: 0,
+      maximumFractionDigits: 3,
+    },
+  )
+
+  function renderHeaderContent() {
+    return (
+      <div className="section header-container">
+        <div className="info-container">
+          <div className="values">
+            <div className="title">{t('amm_lp_token_balance')}</div>
+            <div className="value">{localizedLPBalance}</div>
+          </div>
+          <div className="values">
+            <div className="title">{t('amm_token_balance')}</div>
+            <div className="value">{localizedBalance1}</div>
+          </div>
+          <div className="values">
+            <div className="title">{t('amm_token_balance')}</div>
+            <div className="value">{localizedBalance2}</div>
+          </div>
+          <div className="values">
+            <div className="title">{t('trading_fee')}</div>
+            <div className="value">%{localizedTradingFee}</div>
+          </div>
+        </div>
+      </div>
+    )
+  }
+
+  return (
+    <div className="box account-header">
+      <div className="section box-header">
+        <div className="amm-title">Account ID</div>
+        <h2 className="amm">{accountId}</h2>
+        <div className="currency-pair">
+          <Currency {...balance} shortenIssuer />/
+          <Currency {...balance2} shortenIssuer />
```

Comment:
Don't use spreads if you can help it - our linter doesn't like it, and it's more descriptive to name each variable

Reply:
currency would have to be changed to not use a spread here which would require me to go through and change every location we use currency which is out of scope for this project.

Reply:
You can just pass `<Currency currency={balance2.currency} issuer={balance2.issuer}   shortenIssuer />`

Reply:
updated.